### ISSUE TITLE
Nomination Report Auto-Send Destination Change

### ISSUE THREAD

**Issue by katjonzz** [WorldconVotingSystems/nomnom#69]
The auto-send of the nomination report should go to hugo-admin@glasgow2024.org, not hugo-help@glasgow2024.org, please!

Not finding it in the code or I'd have sent a pull request to change it myself, sorry.

Thanks!

**Comment by balen** [comment]
It is in the file convention.py under the folder glasgow2024

**Comment by offbyone** [comment]
@katjonzz did you solve this? The report destination address is set [in the admin interface](https://nominations.glasgow2024.org/admin/nominate/reportrecipient/).

The address that Henry mentions is part of convention configuration, and is the one that we direct members to email when there are issues with their Hugo ballot.

**Comment by katjonzz** [comment]
Nope! When I click the link you posted for the admin interface here in this ticket, I get this:
Permission Denied
Your user, clyde-0125, does not have access to this feature.
